[2024-11-14T21:09:44.452+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-14T21:09:44.463+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2024-11-14T11:05:00+00:00 [queued]>
[2024-11-14T21:09:44.469+0900] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.first_task scheduled__2024-11-14T11:05:00+00:00 [queued]>
[2024-11-14T21:09:44.469+0900] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-11-14T21:09:44.484+0900] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2024-11-14 11:05:00+00:00
[2024-11-14T21:09:44.486+0900] {standard_task_runner.py:72} INFO - Started process 2377405 to run task
[2024-11-14T21:09:44.489+0900] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'first_task', 'scheduled__2024-11-14T11:05:00+00:00', '--job-id', '264', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpaax57g1f']
[2024-11-14T21:09:44.491+0900] {standard_task_runner.py:105} INFO - Job 264: Subtask first_task
[2024-11-14T21:09:44.523+0900] {task_command.py:467} INFO - Running <TaskInstance: first_dag.first_task scheduled__2024-11-14T11:05:00+00:00 [running]> on host MN
[2024-11-14T21:09:44.591+0900] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T11:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-14T11:05:00+00:00'
[2024-11-14T21:09:44.592+0900] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-14T21:09:44.607+0900] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-14T21:09:44.607+0900] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the fist task!']
[2024-11-14T21:09:44.612+0900] {subprocess.py:99} INFO - Output:
[2024-11-14T21:09:44.613+0900] {subprocess.py:106} INFO - hello world, this is the fist task!
[2024-11-14T21:09:44.613+0900] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-14T21:09:44.632+0900] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-14T21:09:44.632+0900] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=first_task, run_id=scheduled__2024-11-14T11:05:00+00:00, execution_date=20241114T110500, start_date=20241114T120944, end_date=20241114T120944
[2024-11-14T21:09:44.661+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-14T21:09:44.673+0900] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-14T21:09:44.674+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
